pkg/virt-handler/vm.go

```diff
@@ -2641,6 +2641,13 @@ func (d *VirtualMachineController) finalizeMigration(vmi *v1.VirtualMachineInsta
 	if err != nil {
 		return err
 	}
+
+	err = d.podIsolationDetector.AdjustResources(vmi)
```

Comment:
In reality, a failure here doesn't affect the result of migration.
The migration will still show as succeeded. However, perhaps we do need to react differently in this case? 
Maybe, we should at least send an alert in case we didn't manage to plug the devices back.

Reply:
You right, its wrong to return on `AdjustResources` error, thanks for the heads up.
I will log the error instead of returning, then in case a device is failed to plug it will be logged.